db/seeds.rb

```diff
@@ -1,13 +1,9 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the rake db:seed (or created alongside the db with db:setup).
-#
-# Examples:
-#
-#   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
-#   Mayor.create(name: 'Emanuel', city: cities.first)
-Skill.create([{ name: 'skill1', featured: true },
+skills = Skill.create([{ name: 'skill1', featured: true },
               { name: 'skill2', featured: true },
               { name: 'skill3', featured: false },
               { name: 'skill4', featured: true },
               { name: 'skill5', featured: false },
              ])
+
+puts " Successfully added #{skills.count} skills from seed data. ".center(70, '~*~')
```

Comment:
Line is too long. [84/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.